traffic_ops/traffic_ops_golang/cachegroup/dspost.go

```diff
@@ -99,7 +107,7 @@ func postDSes(tx *sql.Tx, user *auth.CurrentUser, cgID int64, dsIDs []int64) (tc
 	if err := updateParams(tx, dsIDs); err != nil {
 		return tc.CacheGroupPostDSResp{}, nil, errors.New("updating delivery service parameters: " + err.Error()), http.StatusInternalServerError
 	}
-	api.CreateChangeLogRawTx(api.ApiChange, fmt.Sprintf("assign servers in cache group %v to deliveryservices %v", cgID, dsIDs), user, tx)
+	api.CreateChangeLogRawTx(api.ApiChange, "CACHEGROUP: "+string(cgName)+", ID: "+strconv.FormatInt(cgID, 10)+", ACTION: Assign DSes to CacheGroup servers", user, tx)
```

Comment:
The requirements of `api.CreateChangeLogBuildMsg` are a little too constrained to use everywhere in the API. For example, the `ACTION:` message can only be in the form of `CRUD verb + object type`. It might be possible to use `api.CreateChangeLog` but this would require implementing the `ChangeLogger` interface for each object type you'd want to use it with, which doesn't sound worth it to me